app/components/point-input.jsx

```diff
@@ -0,0 +1,185 @@
+'use strict'
```

Comment:
I didn't ask for this before, but I'm starting now. Per the style guide, please put a link to the issue as a comment at the top of this file, and of the story.  That way future developers can get back to this content.